jdaviz/tests/test_show.py

```diff
@@ -2,10 +2,12 @@
 
 
 def test_show_popout(specviz_helper):
-    with pytest.raises(RuntimeError, match='Error in displaying Jdaviz'):
+    with pytest.raises(RuntimeError, match='Error in displaying Jdaviz') as err:
         specviz_helper.show(loc="popout")
+        assert isinstance(err.__cause__, NotImplementedError)
```

Comment:
Yes, but let's do that outside of the context manager as pytest recommended in https://docs.pytest.org/en/stable/how-to/assert.html#assertions-about-expected-exceptions 😸 

```suggestion
    assert isinstance(err.__cause__, NotImplementedError)
```